packages/manager/modules/pci/src/projects/project/serving/serving.routing.js

```diff
@@ -0,0 +1,61 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.serving', {
+      url: '/serving',
+      component: 'pciProjectServing',
+
+      redirectTo: transition => Promise.all([
```

Comment:
Please don't use `Promise`, it doesn't mix well with `AngularsJS` 😅 

Instead use `$q.all`

Reply:
26acd4241ae282c2e78a39939cf300f0daa3ca99